docs/dom-testing-library/faq.md

```diff
@@ -76,20 +76,20 @@ in the selector.
 const thirdLiInUl = container.querySelector('ul > li:nth-child(3)')
 ```
 
-Or you could include the index or an ID in your attribute:
+Or you could query the `listitem` role and access the index in question:
 
 ```javascript
-;`<li data-testid="item-${item.id}">{item.text}</li>`
+;`<li>{item.text}</li>`
 ```
 
-And then you could use the `getByTestId` utility:
+And then you could use the `getAllByRole` utility:
```

Comment:
I believe this sentence belongs to the one above. It made sense when we first added the `testid` attribute and then showed how to query it, but since the attribute is gone, I believe the whole content could be collapse into a single paragraph:

```
Or you could use the `getAllByRole` utility querying by 
the `listitem` role and access the index in question:

const items = [ 
  /* your items */  
]
const container = render(/* however you render this stuff */) 
const thirdItem = getAllByRole(container, 'listitem')[2] 
```

thoughts?

Reply:
Absolutely :+1: Thanks.

Reply:
> Or you could use getAllByRole to query the listitem role and access the index in question:

https://deploy-preview-305--testing-library.netlify.com/docs/dom-testing-library/faq